packages/rspack-plugin-html/src/index.ts

```diff
@@ -1220,13 +1209,12 @@ function hookIntoCompiler(
 		try {
 			return require("html-minifier-terser").minify(html, options.minify);
 		} catch (e) {
-			// @ts-expect-error
+			if (!(e instanceof Error)) {
+				throw e;
+			}
 			const isParseError = String(e.message).indexOf("Parse Error") === 0;
 			if (isParseError) {
-				// @ts-expect-error
-				e.message = "html-minifier-terser error:";
-				// @ts-expect-error
-				"\n" + e.message;
+				e.message = "html-minifier-terser error:\n" + e.message;
```

Comment:
It looks like we want to prefix `e.message` here. 

https://github.com/jantimon/html-webpack-plugin/blob/fe231d3d3d256c2bb904b9e0f3f1e7aa67d7f3cd/index.js#L715

@ahabhgk

Reply:
yes